yaml/pois.yaml

```diff
@@ -260,6 +260,9 @@ filters:
   - filter: {tourism: winery}
     min_zoom: GREATEST(14, LEAST(zoom + 2.85, 17))
     output: {kind: winery}
+  - filter: {tourism: gallery}
+    min_zoom: GREATEST(14, LEAST(zoom, 17))
```

Comment:
He requested 15 in #990. I guess I was confused about this notation here. I'll update it to 15.